internal/ci/gui_test

```diff
@@ -2,9 +2,14 @@
 
 set -e
 
+yarn tslint
+yarn lint-ci
+
 cd gui
-yarn tslint && yarn lint-ci && yarn test-ci
+../node_modules/.bin/cross-env NODE_ENV=test \
+  ../node_modules/.bin/jest --runInBand --coverage --reporters jest-silent-reporter
```

Comment:
Perhaps we should just add `../node_modules/.bin` to PATH in this bash script. You could also do `yarn run cross-env ... yarn run jest ...`

Why not just continue to run `yarn test-ci`?

Reply:
CI fails with this message:
```
yarn run v1.12.1
$ cross-env NODE_ENV=test yarn jest --runInBand --coverage --reporters jest-silent-reporter
error Command "jest" not found.
```
It's weird, because jest is installed as a dependency for this repo. I didn't investigate, as it seemed cleaner to move the command directly into the CI script, since it wasn't used anywhere else.